src/Sarif.FunctionalTests/v2/ConverterTestData/FortifyFpr/FortifyTest.fpr.sarif

```diff
@@ -0,0 +1,377 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "HP Fortify Static Code Analyzer"
+      },
+      "invocations": [
+        {
+          "commandLine": "[REMOVED]insourceanalyzer.exe -b CareerGuideAdminTool -verbose -debug -logfile D:\\AgentWork\\574\\a\\sca_artifacts\\CareerGuideAdminTool_scan.log -scan -f D:\\AgentWork\\574\\a\\sca_artifacts\\CareerGuideAdminTool.fpr",
+          "startTime": "2018-06-14T05:51:41.000Z",
+          "machine": "ACESDLFORT00005",
+          "account": "fortbld",
+          "properties": {
+            "Platform": "Windows Server 2012 R2"
+          }
+        }
+      ],
+      "files": {
+        "ContentFileHandlingController.cs": {
+          "length": 58665,
+          "mimeType": "text/x-csharp",
+          "encoding": "utf-8",
+          "properties": {
+            "FprFileType": "csharp"
+          }
+        },
+        "ContentFileViewModel.cs": {
+          "length": 9200,
+          "mimeType": "text/x-csharp",
+          "encoding": "utf-8",
+          "properties": {
+            "FprFileType": "csharp"
+          }
+        },
+        "DocumentValidator.cs": {
+          "length": 51630,
+          "mimeType": "text/x-csharp",
+          "encoding": "utf-8",
+          "properties": {
+            "FprFileType": "csharp"
+          }
+        }
+      },
+      "results": [
+        {
+          "ruleId": "D0ACE9F8-3C8F-42D4-BDB1-2FC9ECB0CDFB",
+          "message": {
+            "text": "<Content><Paragraph>The framework binder used for binding the HTTP request parameters to the model class in <Replace key=\"PrimaryCall.name\"/> on line [12](1) has not been explicitly configured to allow, or disallow, certain attributes.<AltParagraph>The framework binder used for binding the HTTP request parameters to the model class has not been explicitly configured to allow, or disallow, certain attributes.</AltParagraph></Paragraph></Content>"
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "fileLocation": {
+                  "uri": "ContentFileViewModel.cs"
```

Comment:
>ContentFileViewModel. [](start = 26, length = 21)

this output seems to contain file names only, with no directories of any kind. don't we have any enlistment path details in our FPR data?

Reply:
I had full paths in there, which I removed based on the other Fortify test file. I'll put them back.

---
In reply to: [207675249](https://github.com/Microsoft/sarif-sdk/pull/981#discussion_r207675249) [](ancestors = 207675249)